inspirehep/config.py

```diff
@@ -788,7 +788,8 @@
         },
         suggesters=dict(
             journal_title=dict(completion=dict(
-                field='title_suggest'
+                field='title_suggest',
+                size=10
```

Comment:
As it can be seen from this diff, not adding trailing commas means that, whenever we add a new field, we get "dirty" diffs. Adding a trailing comma here will prevent this from happening in a future commit that will modify this configuration.